### PR TITLE
Docs Rebranding 

### PR DESCRIPTION
**Pull request by MarianRaphael**
## Description

Update Docs "FlowForge" -> "FlowFuse"

## Related Issue(s)

https://github.com/flowforge/admin/issues/207

## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/flowforge/flowforge/blob/main/CONTRIBUTING.md)
 - [ ] Suitable unit/system level tests have been added and they pass 
 - [ ] Documentation has been updated
    - [ ] Upgrade instructions
    - [ ] Configuration details
    - [ ] Concepts
 - [ ] Changes `flowforge.yml`?
    - [ ] Issue/PR raised on `flowforge/helm` to update ConfigMap Template
    - [ ] Issue/PR raised on `flowforge/CloudProject` to update values for Staging/Production

## Labels

 - [ ] Backport needed? -> add the `backport` label
 - [ ] Includes a DB migration? -> add the `area:migration` label

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -2,56 +2,56 @@
...
- - [Testing](./contribute/#testing) - Understand our testing philosophy at FlowForge.
+ ## Contributing to FlowFuse
+ - [Useful Information](./contribute/#contributing-to-flowforge) - Learn the foundational concepts of how FlowFuse is built & structured. 
+ - [Development Setup](./contribute/#development-setup) - Configure your local development environment to contribute to FlowFuse.
+ - [Testing](./contribute/#testing) - Understand our testing philosophy at FlowFuse.
  
 ## Support
 
 - [Troubleshooting](./user/#debugging-and-fixing-node-red-issues)
 - [Community Support](https://community.flowforge.com/)
```
Comment (Steve-Mcl) [inline docs/README.md:56]:
```suggestion
- [Community Support](https://community.flowfuse.com/)
```

This is now `https://community.flowfuse.com/`

File: docs/admin/licensing.md [hunk 2]
```diff
@@ -43,9 +43,9 @@ The following features are available in FlowForge Premium:
...
-[^License-Expiry]: The Premium License is valid for 12 months from the date of purchase. The FlowForge Platform will not halt operation when the license expires, but will continue to operate to allow for smooth and continuous operation. Expiry will be logged and reported to the license holder and FlowForge Inc. The license holder will need to renew the license to continue using the platform or revert to the Open-Source version.
-[^intercommunication]: FlowForge Project Nodes provide seamless passing of data and messages between your Node-RED instances.
+[^License-Expiry]: The Premium License is valid for 12 months from the date of purchase. The FlowFuse Platform will not halt operation when the license expires, but will continue to operate to allow for smooth and continuous operation. Expiry will be logged and reported to the license holder and FlowForge Inc. The license holder will need to renew the license to continue using the platform or revert to the Open-Source version.
```
Comment (Steve-Mcl) [inline docs/admin/licensing.md:46]: Just double checking - the use of FlowForge Inc in these 2 sentences are still valid and correct?

File: docs/contribute/README.md [hunk 3]
```diff
@@ -116,8 +116,8 @@ likely want to begin.
 ## Development Setup
 
 1. [Create a Stack](#create-a-stack)
-1. [Running FlowForge](#running-flowforge)
-1. [Configuring FlowForge](#configuring-flowforge)
+1. [Running FlowFuse](#running-flowforge)
+1. [Configuring FlowFuse](#configuring-flowforge)
```
Comment (Steve-Mcl) [inline docs/contribute/README.md:120]:
```suggestion
1. [Running FlowFuse](#running-flowfuse)
1. [Configuring FlowFuse](#configuring-flowfuse)
```

internal links will need updating to match the `### title` changes

File: docs/contribute/README.md [hunk 4]
```diff
@@ -78,7 +78,7 @@ flowforge-dev-env
 More details on using the FlowForge Development Environment are available in its
```
Comment (Steve-Mcl) [inline docs/contribute/README.md:78]: This still refers to `FlowForge Development Environment` however the link to the developer env on github (up at line 49) was renamed to `[FlowForge Development Environment](https://github.com/flowforge/flowforge-dev-env)`

File: docs/install/kubernetes/aws.md [hunk 5]
```diff
@@ -54,7 +54,7 @@ apiVersion: eksctl.io/v1alpha5
 kind: ClusterConfig
 
 metadata:
-  name: flowforge
+  name: FlowFuse
```
Comment (Steve-Mcl) [inline docs/install/kubernetes/aws.md:57]: Just checking. Is this now current?

In an earlier part of the review, we still refer to AWS as using `flowforge` moniker

**Review by Steve-Mcl — CHANGES_REQUESTED** [review]
A handful of questions and some change requests. Not bad considering the size of this PR - phew.

File: docs/install/kubernetes/aws.md (hunk 5 above)
Comment (knolleary) [reply docs/install/kubernetes/aws.md:57]: Not sure this should be changed until we do other packaging changes - @hardillb to confirm.

File: docs/admin/licensing.md (hunk 2 above)
Comment (MarianRaphael) [reply docs/admin/licensing.md:46]: FlowForge Inc. is still the legal entity, so I would keep it as it is for now.

File: docs/contribute/README.md (hunk 4 above)
Comment (MarianRaphael) [inline docs/contribute/README.md:78]:
```suggestion
More details on using the FlowFuse Development Environment are available in its
```

Comment (MarianRaphael) [reply docs/contribute/README.md]: Good catch

File: docs/install/kubernetes/aws.md (hunk 5 above)
Comment (hardillb) [reply docs/install/kubernetes/aws.md:57]: That is just the name of the cluster in AWS, it can be chnaged

**Comment by Steve-Mcl** [comment]
@MarianRaphael good to go. 

I will leave the merge timing to you.

**Pull request merged**